pkgs/top-level/stage.nix

```
@@ -266,6 +266,11 @@ let
       } // lib.optionalAttrs (stdenv.hostPlatform.system == "powerpc64-linux") {
         gcc.abi = "elfv2";
       };
+    } // lib.optionalAttrs stdenv.hostPlatform.isDarwin {
```

It doesn't make sense for `isStatic` to be set on Linux and Darwin but not other operating systems.

```suggestion
    } // lib.optionalAttrs (!stdenv.hostPlatform.isDarwin) {
```